src/main/java/com/example/demo/service/impl/UserServiceImpl.java

```diff
@@ -29,6 +34,12 @@ public UserResponseDto register(UserRegistrationRequestDto request)
 
         User user = userMapper.toEntity(request);
         user.setPassword(passwordEncoder.encode(request.getPassword()));
+
+        // Assign USER role by default
+        Role userRole = roleRepository.findByName(RoleName.ROLE_USER)
+                .orElseThrow(() -> new RuntimeException("Default role USER not found"));
```

Comment:
use your enum USER for message
```suggestion
                .orElseThrow(() -> new EntityNotFoundException("Default role USER not found"));
```